static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-liveblog.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+import { logView } from 'common/modules/commercial/acquisitions-view-log';
+import template from 'lodash/utilities/template';
+import $ from 'lib/$';
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+import ElementInView from 'lib/element-inview';
+import fastdom from 'lib/fastdom-promise';
+import liveblogEpicTemplate from 'raw-loader!common/views/acquisitions-epic-liveblog.html';
+import { liveblog as liveblogCopy } from 'common/modules/commercial/acquisitions-copy';
+
+const pageId = config.page.pageId || '';
+
+let isAutoUpdateHandlerBound = false;
+
+const INSERT_EPIC_AFTER_CLASS = 'js-insert-epic-after';
+
+const getLiveblogEntryTimeData = el => {
```

Comment:
Could you add flow-type information to all these functions? That would be very appreciated!